CMakeLists.txt

```diff
@@ -124,12 +125,14 @@ if(LOVR_USE_LUAJIT AND NOT EMSCRIPTEN)
       target_compile_definitions(buildvm PRIVATE _CRT_SECURE_NO_WARNINGS)
     endif()
     set(LOVR_LUA_INCLUDE deps/luajit/src ${CMAKE_BINARY_DIR}/luajit)
+    set(LOVR_LUA_LIBDIR deps/luajit/src)
```

Comment:
I don't understand why this path (and `deps/lua` for vanilla Lua) are added to the library search path, since there aren't any libraries output in those paths.

Reply:
I guess the intent is so that you can compile the version in `deps` manually and have that library found?

Reply:
It doesn't seem like it would work unless `BEFORE` is added to the `target_link_directories` call though.

Reply:
> I don't understand why this path (and `deps/lua` for vanilla Lua) are added to the library search path, since there aren't any libraries output in those paths.

oh that was just for consistency

Reply:
@bjornbytes fixed